From 6dcd7540d5171e71260f54150ac6b5e43cf75acc Mon Sep 17 00:00:00 2001
From: Nick Zivkovic <nick.zivkovic@joyent.com>
Date: Wed, 15 Feb 2017 23:19:59 +0000
Subject: [PATCH] NAPI-394 update ProvisionNic docs for 'cn_uuid' to mention
 being required for fabric networks (fixed typo) Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Dave Eddy <dave.eddy@joyent.com>

---
 docs/index.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/index.md b/docs/index.md
index f9220d9..e19bdef 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -456,7 +456,7 @@ Creates a new NIC, provisioning an IP and MAC address in the process. The
 | Field             | Type                   | Description                                                                      |
 | ----------------- | ---------------------- | -------------------------------------------------------------------------------- |
 | owner_uuid        | UUID                   | Nic Owner                                                                        |
-| cn_uuid  |        | UUID                   | The UUID of the CN this NIC is on (optional, unless on fabric)                  |
+| cn_uuid           | UUID                   | The UUID of the CN this NIC is on (optional, unless on fabric)                  |
 | belongs_to_uuid   | UUID                   | The UUID of what this Nic belongs to                                             |
 | belongs_to_type   | String                 | The type that this belongs to (eg: 'zone', 'server')                             |
 | ip                | String                 | IP address to assign to the nic                                                  |
-- 
2.21.0

